### ISSUE TITLE
[WCAG] Ensure keyboard navigation - Legend Panel

### ISSUE THREAD

**Issue by kenchase** [Canadian-Geospatial-Platform/geoview#3218]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

Review tab order of all focusable elements within the panel

1 - Add tooltip on button/icon (toggleItemsVisibility) in legend-layer-items.tsx
2 - Check if ListItemButton component is missing an aria-label
